app/i18n/es/admin.php

```diff
@@ -187,33 +187,33 @@
 			),
 			'title' => 'Formulario de registro del usuario',
 		),
-		'sensitive-parameter' => 'Sensitive parameter. Edit manually in <kbd>./data/config.php</kbd>',	// TODO
+		'sensitive-parameter' => 'Parámetro sensible. Lo puedes editar manualmente en <kbd>./data/config.php</kbd>',
 		'tos' => array(
 			'disabled' => 'no se proporciona',
 			'enabled' => '<a href="./?a=tos">está activado</a>',
 			'help' => 'Cómo <a href="https://freshrss.github.io/FreshRSS/en/admins/12_User_management.html#enable-terms-of-service-tos" target="_blank">activar las Condiciones de servicio</a>',
 		),
 		'websub' => array(
-			'help' => 'About <a href="https://freshrss.github.io/FreshRSS/en/users/WebSub.html" target="_blank">WebSub</a>',	// TODO
+			'help' => 'Acerca de <a href="https://freshrss.github.io/FreshRSS/en/users/WebSub.html" target="_blank">WebSub</a>',
 		),
 	),
 	'update' => array(
 		'_' => 'Actualizar sistema',
 		'apply' => 'Aplicar',
-		'changelog' => 'Changelog',	// TODO
+		'changelog' => 'Changelog',	// IGNORE
 		'check' => 'Buscar actualizaciones',
-		'copiedFromURL' => 'update.php copied from %s to ./data',	// TODO
+		'copiedFromURL' => 'update.php copiado desde %s a ./data',
 		'current_version' => 'Dispones de la versión',
 		'last' => 'Última comprobación',
-		'loading' => 'Updating…',	// TODO
+		'loading' => 'Actualizando...',
```

Comment:
```suggestion
		'loading' => 'Actualizando…',
```

Unicode typography

Reply:
In Spanish the simple point is used.

Reply:
Please see https://es.wikipedia.org/wiki/Puntos_suspensivos#Uso_en_computadoras

Reply:
My apologies, this is the first time I know about this. I have learned something new. 

Thank you.

Fixed in 37e1a18 commit.